### PR TITLE
movie: Godzilla (2001) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Twenty-five down, ten to go... Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -211,6 +211,46 @@ cryptid [A scientist explained that the strange insects were actually prehistori
...
+human vs. monster [Yokohama and other places were the scenes of death and mass destruction when the titular monster Godzilla rampaged around Japan.]
+journalism [The "BS Digital Q" media outlet reporter Yuri Tachibana's following of the Godzilla attacking Japan news angle was central to the story.]
+father and daughter [The story follows the reporter Yuri as she convinces her admiral father, Taizo, of the truth of an ancient legend stating that the trio of sacred beasts Mothra, Baguron, and King Ghidorah would come to save Japan from Godzilla.]
+war crimes [It turned out that the souls of those who were killed during the Pacific War were embedded within Godzilla and were lashing out due to modern Japan's denial of its past crimes.]
+female independence [The female reporter Yuri was struggling to be taken seriously in a male dominated media landscape. She ultimately ignored the instructions of her various male superiors at the "BS Digital Q" media outlet and pursue the Godzilla story against their wishes.]
```
[the remark this answers is not in the document]
Comment (odinlake) [reply notes/stories/film/film-scifi-2000s.st.txt:242]: Instinctively I'd say female independence is more likely to be topical in the 50s and before, when women had to obey their husbands etc.

**Review by odinlake — APPROVED** [review]

**Comment by odinlake** [comment]
A comment inline

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:242]: I can't myself justify **equal career opportunities for women** here. It wasn't as if Yuri was passed over for a promotion by a less qualified man or anything like that. I suppose we could just knock up **sexism in society** to major and capture what was going on that way.

**Comment by odinlake** [comment]
If she’s being treated differently in the workplace because she’s a woman
then she’s not having equal opportunity. That’s my take. Not being taken
seriously means not getting promotions. You can change the theme definition
so that it intuitively fits better if you like… I mean what we want is one
umbrella theme for all the problems women face in the workplace relating to
their career advancement (sexual harassment might still be a different
theme).

**Comment by paul-sheridan** [comment]
Alright, let's use **equal career opportunities for women** for now and sort it out later. I effected the change and committed.

**Comment by odinlake** [comment]
Assuming your assessment is that there were problems relating to both
career and gender, I think that’s the right thing to do.

**Pull request merged**